mvcc/backend/backend.go

```diff
@@ -315,6 +316,8 @@ func (b *backend) run() {
 		if b.batchTx.safePending() != 0 {
 			b.batchTx.Commit()
 		}
+		batchIntervalSec.Observe(time.Since(start).Seconds() * 1e3)
```

Comment:
I believe `.Seconds()` already performs the needed unit conversion and we don't need the `* 1e3`.

Reply:
Switched to seconds.